### PR TITLE
Update REPL documentation

### PR DESCRIPTION
**Pull request by ibraheemdev**
The `evcxr_repl` is amazing! However, the readme and documentation is pretty scarce. I updated the readme with codeblocks and more details for some of the useful options. Hopefully this will make the REPL more appealing to viewers.

[View rendered updates](https://github.com/ibraheemdev/evcxr/tree/patch-1/evcxr_repl)

### CODE REVIEW

**Comment by ibraheemdev** [comment]
@googlebot I signed it!

File: evcxr_repl/README.md [hunk 1]
```diff
@@ -1,58 +1,187 @@
...
-* Load crates from crates.io.
-  * e.g. `:dep regex = { version = "1.0" }` will load the regex crate.
-  * This can take a while, especially for large crates with many dependencies.
-* Expressions will be debug printed.
-* For the most part compilation errors are reported in a reasonably intuitive inline way.
+**Define functions, structs, enums, and other data types:**
+```rust
+>> pub struct User {
+     username: String
+  }
```
Comment (bjorn3) [inline evcxr_repl/README.md:34]:
```suggestion
       username: String
   }
```

**Comment by davidlattimore** [comment]
Thanks for the documentation updates @ibraheemdev and thanks for reviewing this and numerous other changes @bjorn3! Unfortunately the Google CLA bot won't let me commit the change, since @bjorn3 hasn't accepted the Google CLA. I realise it's pretty silly given it's a whitespace only change, but I guess it doesn't have any sort of threshold. The CLA, if you accept it would only apply to changes that you contribute to Google projects. If you'd rather not accept it though, I'll delete that commit and we can fix the indentation in a different commit.

**Comment by bjorn3** [comment]
I would prefer not to sign a CLA.

**Comment by davidlattimore** [comment]
No problem. I've removed that commit. Thanks, both of you!

**Pull request merged**